lib/rubocop/cop/style/string_hash_keys.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of strings as keys in hashes. The use of
+      # symbols is preferred instead.
+      #
+      # @example
+      #   # bad
+      #   { 'one' => 1, 'two' => 2, 'three' => 3 }
+      #
+      #   # bad
+      #   { 'one' => 1, two: 2, three: 3 }
+      #
+      #   # good
+      #   { one: 1, two: 2, three: 3 }
+      class StringHashKeys < Cop
+        # TODO: Implement the cop into here.
+        MSG = 'Prefer symbols instead of strings as hash keys.'.freeze
+
+        def_node_matcher :string_hash_key?, <<-PATTERN
+          (pair (str _) _)
+        PATTERN
+
+        def on_pair(node)
+          return unless string_hash_key?(node)
+          add_offense(node)
+        end
+
+        private
+
+        def autocorrect(node)
```

Comment:
If you don't want to implement it in this iteration, just ✂️ it out. Otherwise RuboCop will incorrectly indicate that this cop supports autocorrect. 🙂